dev/buildtool/ga-defaults.yml

```diff
@@ -21,6 +21,7 @@
 bintray_jar_repository: jars
 bintray_org: spinnaker-releases
 bintray_debian_repository: debians
+bintray_publish_wait_secs: -1
```

Comment:
This is correct, but this ga-defaults.yml file is not actually used anywhere.
It is just here for documentation purposes.
You will need to add this to the "buildtool.yml" file on the build server.

Reply:
Yup, that confused me at first, but I figured it out and have already updated that file.